### ISSUE TITLE
Error: ModuleNotFoundError: No module named 'pyecwid.validators'

### ISSUE THREAD

**Issue by ddesmond** [DanPalmz/pyecwid#10]
Hi!

im trying your python library, but at the first example i get this error. I have keys in place (token and shopid)
Error reads:

```
from pyecwid.validators import paramater_validators as validator
ModuleNotFoundError: No module named 'pyecwid.validators'
```

Running python 3.6.5 on OS X.
Thanks!

**Comment by DanPalmz** [comment]
Thanks for the report - I managed to reproduce this using install from Pypi/pip.  Will look at what's going on.  

In the meantime if you're keen to test:
I didn't have the same issue using the tarball here https://github.com/DanPalmz/pyecwid/releases/tag/0.1.0 (just grab the inner pyecwid directory and place it in your project and imports should work for now..)

**Comment by DanPalmz** [comment]
Have released 0.1.1 which fixes this - should be on Pypi now.

Still getting my head around python imports - I was calling the validators in a subdirectory without initialisting it as a subpackage - worked fine when it was local but not an imported package.   Thanks for reporting the bug - was a simple one but major!

**Comment by ddesmond** [comment]
gg! thanks !! il try too work with and let you know! ;)
